### PR TITLE
Assignment 2 completed

### PR DESCRIPTION
**Pull request by ansul90**
Added journal file too

### CODE REVIEW

**Review by vramana — CHANGES_REQUESTED** [review]
The journal file should look like this 
https://raw.githubusercontent.com/fcc-hyd/moc-android/master/sampleJournal.md

File: ansul/Assignment 1/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    <EditText
+        android:id="@+id/username"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_centerHorizontal="true"
+        android:layout_centerVertical="true"
+        android:background="@android:color/white"
```
Comment (mallelapavank) [inline ansul/Assignment 1/activity_main.xml:17]: you've set the background color to white but your text color is also white in this theme which makes the entered text invisible. please change your text color to black or any other contrast color.
consider this change in your constraint layout that you will make next. dont make any changes here. I'm merging this now.

Comment (mallelapavank) [inline ansul/Assignment 1/activity_main.xml]: consider this change in your constraint layout that you will make next. dont make any changes here. I'm merging this now.

File: ansul/journal.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# ansul90 Log File | MOC-ANDROID
+
+### 1/6/17
+
+Today learnt about layouts
```
Comment (mallelapavank) [inline ansul/journal.md:5]: like @vramana suggested your journal should be in this format: if any questions ask on gitter chatroom
https://raw.githubusercontent.com/fcc-hyd/moc-android/master/sampleJournal.md

**Pull request closed without merging**